server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tables/TableInformation.java

```diff
@@ -30,7 +29,7 @@
 
   // Variable names become JSON keys
   public String tablename;
-  public Table.ID tableId;
+  public String tableId;
```

Comment:
Could change the variable name, like you did below.  You would probably have to change it on the front end as well though.  You could use tableIdString if the other is too long.  It is just nice to know you are working with a string and not the Table.ID object.